corehq/messaging/scheduling/urls.py

```diff
@@ -2,10 +2,14 @@
 
 from corehq.messaging.scheduling.views import (
     BroadcastListView,
+    CreateMessageView,
+    EditMessageView,
     possible_sms_recipients,
 )
 
 urlpatterns = [
     url(r'^broadcasts/$', BroadcastListView.as_view(), name=BroadcastListView.urlname),
+    url(r'^broadcasts/add/$', CreateMessageView.as_view(), name=CreateMessageView.urlname),
+    url(r'^broadcasts/edit/(?P<broadcast_id>[\w-]+)/$', EditMessageView.as_view(), name=EditMessageView.urlname),
```

Comment:
i'm not sure if i like the name of CreateMessage / EditMessage in the views, the templates, and the js files, since we're really scheduling abstract content (which could also be a survey, ivr survey, etc.).  can we rename the associated files to  create schedule and edit schedule?

Reply:
https://github.com/dimagi/commcare-hq/pull/17722/commits/a9f1dca7096f85fe921b5010692a5aec6999ef2b